### PR TITLE
testing/nitro: new aport

### PR DESCRIPTION
**Pull request by chambbj**
https://github.com/hobu/nitro
Nitro is a library for reading/writing NITF files.

### CODE REVIEW

File: testing/nitro/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Maintainer: Bradley J Chambers <brad.chambers@gmail.com>
+pkgname=nitro
+pkgver=2.7
+_pkgver="$pkgver"dev-1
+pkgrel=0
+pkgdesc="The Nitro NITF project"
+url="https://github.com/hobu/nitro"
+arch="all"
+license="GPL3"
+depends="libstdc++ libgcc musl"
```
Comment (ncopa) [inline testing/nitro/APKBUILD]: Are those dependencies really needed? They should be automatically detected.

Comment (ncopa) [reply testing/nitro/APKBUILD]: They are automatically detected:
https://travis-ci.org/alpinelinux/aports/builds/252150974#L7304-L7306

Comment (chambbj) [reply testing/nitro/APKBUILD]: Thanks for catching this. I'll make the change.

**Comment by chambbj** [comment]
@ncopa We should be good now unless you prefer to have the entire `depends=""` line removed when not needed.

**Pull request closed without merging**